### ISSUE TITLE
host.docker.internal not resolvable on new WSL installations

### ISSUE THREAD

**Issue by skython** [rancher-sandbox/rancher-desktop#6272]
### Actual Behavior

It looks like Microsoft renamed the WSL network adapter name like this:
`vEthernet (WSL) -> vEthernet (WSL (Hyper-V firewall))`

As a result my Rancher Desktop installation wasn't able to determine the IP for the adapter anymore and wrote something like this into the /etc/hosts file of the rancher-desktop WSL:
```
# BEGIN Rancher Desktop configuration.
undefined    host.rancher-desktop.internal host.docker.internal
# END Rancher Desktop configuration.
```

As a workaround its possible to rename the network adapter in an Admin Powershell using the following command and restart Rancher Desktop afterwards:
`Rename-NetAdapter -name "vEthernet (WSL (Hyper-V firewall))" -newname "vEthernet (WSL)" -IncludeHidden`

### Steps to Reproduce

Do a fresh new WSL installation or rename the WSL adapter using the following command:
`Rename-NetAdapter -name "vEthernet (WSL)" -newname "vEthernet (WSL (Hyper-V firewall))" -IncludeHidden`

### Result

The host names `host.rancher-desktop.internal` and `host.docker.internal` are not resovable anymore

### Expected Behavior

Rancher Desktop should find the interface with the new name. Maybe there is some other unique property to that interface or matching can be done by something like  `startsWith("vEthernet (WSL")`

### Rancher Desktop Version

1.11.1

### Which container engine are you using?

moby (docker cli)

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 11 22H2

### What CPU architecture are you using?

x64

**Comment by Nino-K** [comment]
I believe this is the similar issue as this: https://github.com/rancher-sandbox/rancher-desktop/issues/5984

**Comment by jandubois** [comment]
Need to verify if this is fixed by #6933.

**Comment by jandubois** [comment]
> I believe this is the similar issue as this: #5984

Yes, it is a duplicate.

> Need to verify if this is fixed by #6933.

It does indeed seem to be fixed as expected, so will be in 1.14.0
